### PR TITLE
Fix errorf conventions

### PR DESCRIPTION
**Pull request by lumjjb**
Signed-off-by: Brandon Lum <lumjjb@gmail.com>

#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

Follow a single convention of using `github.com/pkg/errors` instead of `fmt` for errors.

#### How to verify it

Run to check no files:
`grep -r "fmt\.Errorf" * | grep -v vendor`

#### Which issue(s) this PR fixes:

NONE

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests.
Nice set of changes, tyvm!

File: run_test.go [hunk 1]
```diff
@@ -60,15 +59,15 @@ func TestAddRlimits(t *testing.T) {
 				for _, rlimit := range rlimits {
 					if rlimit.Type == "RLIMIT_FSIZE" {
 						if rlimit.Hard != 4096 {
-							return fmt.Errorf("expected spec to have %#v hard limit set to %v but got %v", rlimit.Type, 4096, rlimit.Hard)
+							return errors.Errorf("expected spec to have %#v hard limit set to %v but got %v", rlimit.Type, 4096, rlimit.Hard)
```
Comment (TomSweeneyRedHat) [inline run_test.go:62]: lint is unhappy with this line.  Could it be errors doesn't handle `%#v`?

Comment (lumjjb) [reply run_test.go:63]: Fixed, was a bad (missing) import. repushed

**Comment by rhatdan** [comment]
@lumjjb I guess we were consistently incosistent.
LGTM
bors r+

**Pull request merged**